### ISSUE TITLE
OSX Failure To Start

### ISSUE THREAD

**Issue by zoranSpare** [apache/netbeans#6030]
### Apache NetBeans version

Apache NetBeans 18

### What happened

When I try and start the Netbeans App on many occasions the startup will fail. I will have to retry a number of times before it starts.

When I try starting from the commandline I get the following error:
```
java.lang.NullPointerException
	at java.base/java.net.URLConnection$1.getContentTypeFor(URLConnection.java:313)
	at java.base/java.net.URLConnection.guessContentTypeFromName(URLConnection.java:1464)
	at java.base/sun.net.www.protocol.jar.JarURLConnection.getContentType(JarURLConnection.java:223)
	at java.base/java.net.URLConnection.getContentHandler(URLConnection.java:1277)
	at java.base/java.net.URLConnection.getContent(URLConnection.java:778)
	at java.base/java.net.URL.getContent(URL.java:1150)
	at org.openide.util.lookup.MetaInfServicesLookup.search(MetaInfServicesLookup.java:293)
	at org.openide.util.lookup.MetaInfServicesLookup.beforeLookup(MetaInfServicesLookup.java:132)
	at org.openide.util.lookup.MetaInfServicesLookup.beforeLookupResult(MetaInfServicesLookup.java:111)
	at org.openide.util.lookup.AbstractLookup.lookup(AbstractLookup.java:458)
	at org.openide.util.lookup.ProxyLookup$R.initResults(ProxyLookup.java:448)
	at org.openide.util.lookup.ProxyLookup$R.myBeforeLookup(ProxyLookup.java:735)
	at org.openide.util.lookup.ProxyLookup$R.computeResult(ProxyLookup.java:611)
	at org.openide.util.lookup.ProxyLookup$R.allItems(ProxyLookup.java:595)
	at org.openide.util.lookup.ProxyLookup$R.collImpl(ProxyLookup.java:698)
	at org.openide.util.lookup.ProxyLookup$R.collectFires(ProxyLookup.java:644)
	at org.openide.util.lookup.ProxyLookup.setLookups(ProxyLookup.java:227)
	at org.openide.util.lookup.ProxyLookup.setLookups(ProxyLookup.java:183)
	at org.netbeans.core.startup.MainLookup.changeLookups(MainLookup.java:193)
	at org.netbeans.core.startup.MainLookup.doInitializeLookup(MainLookup.java:183)
	at org.netbeans.core.startup.MainLookup.modulesClassPathInitialized(MainLookup.java:134)
	at org.netbeans.core.startup.MainLookup.modulesClassPathInitialized(MainLookup.java:127)
	at org.netbeans.core.startup.NbInstaller.loadImpl(NbInstaller.java:360)
	at org.netbeans.core.startup.NbInstaller.access$000(NbInstaller.java:77)
	at org.netbeans.core.startup.NbInstaller$1.run(NbInstaller.java:322)
	at org.openide.filesystems.FileUtil$2.run(FileUtil.java:420)
	at org.openide.filesystems.EventControl.runAtomicAction(EventControl.java:102)
	at org.openide.filesystems.FileSystem.runAtomicAction(FileSystem.java:494)
	at org.openide.filesystems.FileUtil.runAtomicAction(FileUtil.java:404)
	at org.openide.filesystems.FileUtil.runAtomicAction(FileUtil.java:424)
	at org.netbeans.core.startup.NbInstaller.load(NbInstaller.java:319)
	at org.netbeans.ModuleManager.doEnable(ModuleManager.java:1488)
	at org.netbeans.ModuleManager.enable(ModuleManager.java:1331)
	at org.netbeans.ModuleManager.enable(ModuleManager.java:1256)
	at org.netbeans.core.startup.ModuleList.installNew(ModuleList.java:316)
	at org.netbeans.core.startup.ModuleList.trigger(ModuleList.java:252)
	at org.netbeans.core.startup.ModuleSystem.restore(ModuleSystem.java:298)
	at org.netbeans.core.startup.Main.getModuleSystem(Main.java:173)
	at org.netbeans.core.startup.Main.getModuleSystem(Main.java:142)
	at org.netbeans.core.startup.Main.start(Main.java:303)
	at org.netbeans.core.startup.TopThreadGroup.run(TopThreadGroup.java:98)
	at java.base/java.lang.Thread.run(Thread.java:834)
```
My environment is :
MacOS 13.4 Venture
Java_Home : /Library/Java/JavaVirtualMachines/jdk-11.0.2.jdk/Contents/Home

### How to reproduce

This will happen regularly throughout the day. I would say 2 out of 3 starts fails.

### Did this work correctly in an earlier version?

Apache NetBeans 14

### Operating System

MacOS 13.4 Ventura

### JDK

/Library/Java/JavaVirtualMachines/jdk-11.0.2.jdk/Contents/Home

### Apache NetBeans packaging

Apache NetBeans provided installer

### Anything else

Regularly.

### Are you willing to submit a pull request?

Yes

**Comment by neilcsmith-net** [comment]
> Java_Home : /Library/Java/JavaVirtualMachines/jdk-11.0.2.jdk/Contents/Home

That's an old version of Java 11!  Latest is 11.0.19.

Unless there's a reason you need Java 11, please try an up-to-date JDK.  If you need Java 11, please update to the latest patched version.  Then see if the issue still occurs.

You could also try the community installer at https://www.codelerity.com/netbeans/ which bundle a local JDK 20 and do not rely on any system installed JDK to run the IDE (although you can use them for projects).

**Comment by matthiasblaesing** [comment]
Something similar is reported against visual VM:

https://github.com/oracle/visualvm/issues/467

@thurka found that the problem is present until 11.0.4 and fixed from 11.0.5.

**Comment by neilcsmith-net** [comment]
Thanks @matthiasblaesing - that link takes us back to this being a duplicate of #3958 and #4652 then.
